### PR TITLE
CMF-9: Add MenuManager

### PR DESCRIPTION
**Pull request by CrooZER**

### CODE REVIEW

File: Entity/EntityManager/BaseMenuManager.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Grossum\MenuBundle\Entity\EntityManager;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Grossum\CoreBundle\Entity\EntityTrait\SaveUpdateInManagerTrait;
```
Comment (aivus) [inline Entity/EntityManager/BaseMenuManager.php:7]: Please, saparate this namespaces

File: Entity/EntityManager/BaseMenuManager.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    private $repository;
+
+    /** @var  ObjectManager */
+    private $objectManager;
+
+    public function __construct(ObjectManager $objectManager)
+    {
+        $this->objectManager = $objectManager;
+        $this->repository    = $objectManager->getRepository('GrossumMenuBundle:BaseMenu');
+    }
```
Comment (aivus) [inline Entity/EntityManager/BaseMenuManager.php:49]: Please, add `getRepository` method

**Comment by aivus** [comment]
Please, use the next commit structure:

```
Ticket number: Ticket title
 - list of your changes
 - another change
 - something else
```

Like a:

```
CMF-9: Add MenuManager
 - add manager for BaseMenu entity
```

Also, please fill the PR title like a ticket title.

File: Resources/config/routing.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#ONLY FOR TEST. NEEDS REMOVE AT THE END
+#ADD THIS ROUTING TO MAIN ROUTING CONFIG
+grossum_index:
+    path:      /contact
+    defaults:  { _controller: GrossumMenuBundle:Default:index }
```
Comment (WhiteEagle88) [inline Resources/config/routing.yml]: Please, add new line

File: Manager/MenuManager.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        $this->handlers[$menuHandler->getIdentifierClass()] = $menuHandler;
+    }
+
+    /**
+     * @param string $className
+     * @return MenuHandlerInterface
+     */
+    public function getMenuHandler($className)
+    {
+        if (!array_key_exists((string)$className, $this->handlers)) {
```
Comment (WhiteEagle88) [inline Manager/MenuManager.php]: Why are you strictly convert into a string?

Comment (malavsky) [inline Manager/MenuManager.php]: @WhiteEagle88 because it's php. When convert to string, I'm sure that there will be a string.

Comment (WhiteEagle88) [inline Manager/MenuManager.php]: But in https://github.com/GrossumUA/MenuBundle/pull/2/files#diff-2eda2f5c6527c6f9d8cad045ebf0e8a7R29 you expect string

**Comment by aivus** [comment]
Backmerge required

**Pull request merged**